airflow-core/src/airflow/ui/src/i18n/locales/de/common.json

```
@@ -163,22 +177,40 @@
     "to": "Zu"
   },
   "task": {
+    "documentation": "Task Dokumentation",
     "lastInstance": "Letzte Task Instanz",
     "operator": "Operator",
     "triggerRule": "Auslöse-Regel"
   },
   "task_one": "Task",
   "task_other": "Tasks",
+  "taskId": "Task ID",
   "taskInstance": {
     "dagVersion": "Dag Version",
+    "executor": "Ausführungsumgebung",
+    "executorConfig": "Konfiguration der Ausführungsumgebung",
+    "hostname": "Hostname",
+    "maxTries": "Maximale Versuche",
     "pid": "PID",
     "pool": "Pool",
-    "queuedAt": "Wartend seit"
+    "poolSlots": "Pool Belegung",
+    "priorityWeight": "Priorität",
+    "queue": "Warteschlange",
+    "queuedWhen": "Wartend seit",
+    "scheduledWhen": "Geplant ab",
+    "triggerer": {
+      "assigned": "Zugewiesener Abrufumgebung",
+      "class": "Abruf-Klasse",
+      "createdAt": "Zeitpunkt der Erstellung",
+      "id": "Auslösungs ID",
```

```suggestion
      "id": "Abrufungs ID",
```

Ein Vorschlag, ich denke wir sollten nicht auch "Auslösung" verwenden hier, weil das Wort schon für DAG Triggern verwendet wird 🤔

---

Klingt etwas schräg aber ich übernehme mal Deinen Vorschlag.